.github/workflows/bench-scipp.yml

```diff
@@ -24,22 +24,24 @@ jobs:
         shell: bash -l {0}
         run: git clone https://github.com/scipp/scipp repo
 
+      - run: python repo/tools/metatoenv.py --dir=repo/conda --env-file=buildenv.yml --channels=conda-forge
+
       - name: Cache conda pkgs
         uses: actions/cache@v2
         env:
-          # Increase this value to reset cache if etc/example-environment.yml has not changed
+          # Increase this value to reset cache if buildenv.yml has not changed
           CACHE_NUMBER: 0
         with:
           path: ~/conda_pkgs_dir
           key:
-            ${{ runner.os }}-conda-${{ env.CACHE_NUMBER }}-${{ hashFiles('repo/scipp-developer.yml') }}
+            ${{ runner.os }}-conda-${{ env.CACHE_NUMBER }}-${{ hashFiles('buildenv.yml') }}
 
       - name: Setup Mambaforge and Scipp dev env
         uses: conda-incubator/setup-miniconda@v2.1.1
         with:
           miniforge-version: latest
           miniforge-variant: Mambaforge
-          environment-file: repo/scipp-developer.yml
+          environment-file: buildenv.yml
           activate-environment: scipp-developer
```

Comment:
Not sure if this is an issue, because looking at the logs it all looked fine, apart from the failure at the end, but by default the `name:` field in the generated `buildenv.yml` file will be `buildenv`.
So i don't know if this means you cannot `conda activate scipp-developer` because it creates an env with the name `buildenv`?

Reply:
Isn't the name of the env still `scipp-developer`? Or is it taken from the filename?

Reply:
taken from the filename by default, but you can set it with `--env-name=scipp-developer.yml`